<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>GradientMapConfigWidget</class>
 <widget class="QWidget" name="GradientMapConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gradient Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item column="0" row="0">
      <widget class="QLabel" name="gradientLabel">
       <property name="text">
        <string>Gradient:</string>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="KoColorPopupButton" name="btnGradientChooser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KisStopGradientEditor" name="gradientEditor" native="true"/>
       </item>
      </layout>
     </item>
     <item column="0" row="1">
      <widget class="QLabel" name="colorModeLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Color Mode:</string>
       </property>
       <property name="buddy">
        <cstring>colorModeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item column="1" row="1">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QComboBox" name="colorModeComboBox">
         <item>
          <property name="text">
           <string>Blend</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nearest</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dither</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="colorModeStackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="prefix" stdset="0">
          <string>Amount:  </string>
         </property>
         <widget class="QWidget" name="blendPage">
          <layout class="QFormLayout" name="formLayout_7"/>
         </widget>
         <widget class="QWidget" name="nearestPage">
          <layout class="QFormLayout" name="formLayout_6"/>
         </widget>
         <widget class="QWidget" name="ditherPage">
          <layout class="QFormLayout" name="formLayout_5">
           <item colspan="2" column="0" row="0">
            <widget class="KisDitherWidget" name="ditherWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoColorPopupButton</class>
   <extends>QToolButton</extends>
   <header>KoColorPopupButton.h</header>
  </customwidget>
  <customwidget>
   <class>KisStopGradientEditor</class>
   <extends>QWidget</extends>
   <header>kis_stopgradient_editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisDitherWidget</class>
   <extends>QWidget</extends>
   <header location="global">KisDitherWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>colorModeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>colorModeStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorModeStackedWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>colorModeComboBox</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>